tests/kernel/version_header/src/main.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2012-2014 Wind River Systems, Inc.
```

Comment:
Did someone at Wind River write this test?

Reply:
ok